NAnt.Extensions/Functions/Microsoft/Signing.cs

```diff
@@ -49,12 +53,40 @@ public static bool IsSigned(string filePath)
             if (!File.Exists(filePath)) throw new FileNotFoundException(filePath);
```

Comment:
Look at and use the following:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/get-authenticodesignature?view=powershell-3.0

We can use powershell (3.0+) and remove our dependence on signtool and an SDK version path altogether.  This satisfies our OS requirements for our build nodes.

Sample code:
https://stackoverflow.com/questions/24060009/checking-digital-signature-on-exe

using (PowerShell ps = PowerShell.Create())
            {
                ps.AddCommand("Get-AuthenticodeSignature", true);
                ps.AddParameter("FilePath", calculatedFullPath);
                var cmdLetResults = ps.Invoke();

                foreach (PSObject result in cmdLetResults)
                {
                    Signature s = (Signature)result.BaseObject;
                    isSigned = s.Status.Equals(SignatureStatus.Valid);
                    if (isSigned == false)
                    {
                        ErrorList.Add(string.Format("!!!AuthenticodeSignature status is '{0}' for file '{1}' !!!", s.Status.ToString(), calculatedFullPath));
                    }
                    else
                    {
                        Log.LogMessage(string.Format("!!!AuthenticodeSignature status is '{0}' for file '{1}' !!!", s.Status.ToString(), calculatedFullPath));
                    }
                    break;
                }
            }